osf/management/commands/email_all_users.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+# This is a management command, rather than a migration script, for two primary reasons:
+#   1. It makes no changes to database structure (e.g. AlterField), only database content.
+#   2. It takes a long time to run and the site doesn't need to be down that long.
+
+from __future__ import unicode_literals
+import logging
+
+
+import django
+django.setup()
+
+from django.core.management.base import BaseCommand
+from framework.celery_tasks import app as celery_app
+from framework import sentry
+
+from website import mails
+
+from osf.models import OSFUser
+
+logger = logging.getLogger(__name__)
+
+OFFSET = 50000
+
+@celery_app.task(name='management.commands.email_all_users')
+def email_all_users(email_template, dry_run=False, ids=[], run=0, offset=OFFSET):
```

Comment:
Prefer [immutable default args](https://towardsdatascience.com/6-typical-errors-python-newbie-should-know-39eac4886375#8278). Since `None` and `[]` are both falsey, shouldn't require changes in the function, only signature.